src/screens/SearchScreen.js

```diff
@@ -6,8 +6,16 @@ import Grid from "@mui/material/Grid";
 import LocationSearch from "../components/LocationSearch";
 import Api from "../api/Api";
 import Typography from "@mui/material/Typography";
+import { LoadScript, useJsApiLoader } from "@react-google-maps/api";
+import "./SearchScreen.css";
 
 const SearchScreen = () => {
+  const { isLoaded } = useJsApiLoader({
```

Comment:
move this to separate JS file, so that it can be mocked. This will make the test cases not call google.

Reply:
Updated